src/components/AssetInput/SingleAsset/SelectedAsset.tsx

```diff
@@ -39,13 +39,17 @@ export const SelectedAsset: FC<Required<SelectedAssetProps>> = ({ asset, size, a
 
     const [flyoutWidth, setFlyoutWidth] = useState(0);
 
-    const calcFlyoutWidth = () => {
-        setFlyoutWidth(buttonRef.current?.getBoundingClientRect().width || 0);
-    };
-
     useEffect(() => {
-        calcFlyoutWidth();
-        window.addEventListener("resize", calcFlyoutWidth, false);
+        const calcFlyoutWidth = () => {
+            setFlyoutWidth(buttonRef.current?.getBoundingClientRect().width || 0);
```

Comment:
Might require the setFlyoutWidth to be wrapped in a setTimeout, see the useScrollWrapper hook for an example and explanation. If you try to run a cypress test in clarify that contains the flyout it may or may not fail without it, so worth checking out.

Reply:
Thanks for the heads-up, for now the tests seem to run fine, but to make sure and avoid errors in the future I added this timeout anyway.